src/app/input_manager.ts

```diff
@@ -551,6 +551,8 @@ function getDescriptionForCharacter(character: Character): string[] {
 function getDescriptionForClass(characterClassType: ClassType): string[] {
     const characterClass = getCharacterClassForType(characterClassType);
     return [
+        `<b>${characterClassType}</b>`,
```

Comment:
Drop the `<b>` - no markdown in canvas text

Reply:
Done!